UI.tsv

```diff
@@ -568,62 +568,62 @@ UI_20150317_000567	{@st41}Damage Top 100{/}
 UI_20150317_000568	{@st41}Money Money Top 100{/}
 UI_20150317_000569	{@st41}Level Up Top 100{/}
 UI_20150317_000570	{@st41b}Menu{/}
-UI_20150317_000571	{@st43}{s22}Enhance Item{/}
+UI_20150317_000571	{@st43}{s22}Refine Item{/}
 UI_20150317_000572	{@st41b}Selected Item
 UI_20150317_000573	{@st41b}Anvil
 UI_20150317_000574	{@st41}{s20}Anvil Hit Count :{@st43} %s
 UI_20150317_000575	{@st41} {s20}Price : %s
-UI_20150317_000576	{@st41b}Enhance{/}
-UI_20150317_000577	{@st59}Start Item Enhancement{/}
-UI_20150317_000578	{@st59}Close Enhancement{/}
-UI_20150317_000579	{@st41b} Would you like to enhance the {@sti9}×%s{/} selected items?{/}
-UI_20150317_000580	{memo X}{@st43}Enhance
-UI_20150317_000581	{@st44}Enhance Item{/}
-UI_20150317_000582	{s24}{b}{#000000}Enhancement Completed!
-UI_20150317_000583	{@st41}Enhance Again{/}
+UI_20150317_000576	{@st41b}Refine{/}
+UI_20150317_000577	{@st59}Start the Item Refinement{/}
+UI_20150317_000578	{@st59}Close Refining{/}
+UI_20150317_000579	{@st41b} Would you like to refine the {@sti9}×%s{/} selected items?{/}
+UI_20150317_000580	{memo X}{@st43}Refine
+UI_20150317_000581	{@st44}Refine Item{/}
+UI_20150317_000582	{s24}{b}{#000000}Refining Complete!
+UI_20150317_000583	{@st41}Refine Again{/}
 UI_20150317_000584	{@st41}Close{/}
 UI_20150317_000585	{s18}{ol}{gr gradation1}Repair Equipment{/}
 UI_20150317_000586	Equipped Equipment
-UI_20150317_000587	Carried Equipment
+UI_20150317_000587	Equipment in the Inventory
 UI_20150317_000588	{Gr gradation1}Select All{/}
 UI_20150317_000589	Select All
 UI_20150317_000590	{Gr gradation1}Repair{/}
 UI_20150317_000591	{@st43}Repair{/}
-UI_20150317_000592	{memo X}List of items for repair
+UI_20150317_000592	{memo X}List of repairable items
 UI_20150317_000593	{@st41b}Repair Cost{/}
 UI_20150317_000594	{@st41b}After Repair{/}
-UI_20150317_000595	{@st41b}Repair Equipment{/}
+UI_20150317_000595	{@st41b}Repair{/}
 UI_20150317_000596	Repair the selected equipment.
 UI_20150317_000597	{@st59}Close Item Repair{/}
-UI_20150317_000598	{@st43}Repair Item{/}
-UI_20150317_000599	{@st41}List of items for Repair{/}
-UI_20150317_000600	{@st44}Item Repair{/}
-UI_20150317_000601	{@st42}Repair{/}
-UI_20150317_000602	{@st59}Repair as current settings{/}
-UI_20150317_000603	{@st59}Do not repair{/}
+UI_20150317_000598	{@st43}Item Strengthening{/}
```

Comment:
Updated accordingly based on enquiry response from IMC Games. It's neither servicing nor maintenaning. It is actually the act of strengthening the item beyond its original power without change in appearance. Alternative is augment, harden.